modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/connectivity/oauth/OAuthHandler.java

```diff
@@ -122,8 +129,14 @@ protected Function<OAuthConfig, ObjectStore> buildObjectStoreLocator() {
       String storeName = storeConfig
           .map(OAuthObjectStoreConfig::getObjectStoreName)
           .orElse(BASE_PERSISTENT_OBJECT_STORE_KEY);
-
-      return objectStoreManager.getObjectStore(storeName);
+      ObjectStore os = objectStoreManager.getOrCreateObjectStore(storeName, ObjectStoreSettings
+          .builder()
+          .entryTtl(0L)
```

Comment:
What is the reason for using this default settings?

Reply:
I just wanted to go with some defaults and don't want to add more system properties, may be will change to ObjectStoreSettings.builder().persistent(true).build()